src/main/java/com/swapnixi/feedbackhub/service/implementation/UserServiceImpl.java

```diff
@@ -19,10 +23,24 @@ public class UserServiceImpl implements UserService {
 
     @Override
     public User createUser(User user) {
-//        User user = DtoToUser(user);
-        User savedUser = userRepository.save(user);
-//        return userToDTO(savedUser);
-        return  savedUser;
```

Comment:
Before this API registers the new user without checking whether the user is already registered or not which causes redundancy problem during login.